test/Libraries/AnalysisTests/AnalysisTests.csproj

```diff
@@ -68,6 +83,10 @@
       <Project>{76686ed6-e759-4772-81c2-768740be13fa}</Project>
       <Name>Analysis</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\..\src\Libraries\CoreNodeModels\CoreNodeModels.csproj">
+      <Project>{D8262D40-4880-41E4-91E4-AF8F480C8637}</Project>
+      <Name>CoreNodeModels</Name>
```

Comment:
@RobertGlobant20 - this change is causing atleast some of the 600 failures you are getting.
Change this to `<Private>False<Private>` or in visual studio CopyLocal False, same thing.

This binary is being loaded twice, once from the /nodes folder, and now again from the /bin folder (in your PR). It should only be present in the /nodes folder.

To be safe, I would make all of these dependencies copy Local false, but thats probably not the problem.

Reply:
@mjkkirschner I removed the CoreNodeModels reference from the AnalysisTests project, then there shouldn't be any problem now.
Thanks